src/main/java/ua/rent/masters/easystay/service/impl/TelegramNotificationService.java

```diff
@@ -0,0 +1,37 @@
+package ua.rent.masters.easystay.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import ua.rent.masters.easystay.dto.NotifyResponse;
+import ua.rent.masters.easystay.handler.TelegramBotHandler;
+import ua.rent.masters.easystay.model.Accommodation;
+import ua.rent.masters.easystay.model.Booking;
+import ua.rent.masters.easystay.model.Payment;
+import ua.rent.masters.easystay.model.User;
+import ua.rent.masters.easystay.service.NotificationService;
+
+@Service
+@RequiredArgsConstructor
+public class TelegramNotificationService implements NotificationService {
+    private final TelegramBotHandler botHandler;
+
+    @Override
+    public void notifyAboutAccommodation(Accommodation accommodation, User user) {
+
+    }
+
+    @Override
+    public void notifyAboutBooking(Booking booking, User user) {
+
+    }
+
+    @Override
+    public void notifyAboutPayment(Payment payment, User user) {
+
+    }
```

Comment:
It seams to me that it might be good idea to have different methods to notify accomodation owner and renter.
Owner will receive smth like "user Taras Fedak booked your accommodation with address blablabla"
Renter something like "Your booking is successful"